### DISCUSSION TITLE
How i can use table builder with collection instead query?

### DISCUSSION THREAD

**Discussion by kaozaza2** [filamentphp/filament#7768] [General]
I need to use collection from http request to data on table, how I can do that?

**Comment by danharrin** [comment]
https://usesushi.dev/

**Reply by RPotkowaVegasystems** [reply]
How can i use the TrashedFilter on a Sushi Model? I get 

"Call to undefined method Illuminate\Database\Eloquent\Builder::withoutTrashed()"

**Reply by sinnbeck** [reply]
Does the model have `use SoftDeletes` ?

**Reply by RPotkowaVegasystems** [reply]
Yes

**Reply by RPotkowaVegasystems** [reply]
Ahhh, ok. I mean No. I found it? The trait was missing! Didnt saw it. Thx

**Comment by pmcfernandes** [comment]
See [https://filamentphp.com/content/leandrocfe-how-to-consume-an-external-api-with-filament-tables](https://filamentphp.com/content/leandrocfe-how-to-consume-an-external-api-with-filament-tables)
